python/vyos/ifconfig/interface.py

```diff
@@ -1256,7 +1256,8 @@ def update(self, config):
         tmp = dict_search('ipv6.address.eui64_old', config)
         if tmp:
             for addr in tmp:
-                self.del_ipv6_eui64_address(addr)
+                if addr != '':
```

Comment:
The check should be moved inside the `del_ipv6_eui64_address()` method so all consumers benefit from it.

Reply:
Done

Reply:
That solution is even better! Thanks! 

Can you please also cherry-pick this to current?